app/feature/feature-profile/src/main/kotlin/com/hedvig/android/feature/profile/tab/ProfileDestination.kt

```diff
@@ -228,24 +299,14 @@ private fun ColumnScope.ProfileItemRows(
     icon = Icons.Hedvig.ContactInformation,
     onClick = showMyInfo,
   )
-  AnimatedVisibility(
-    visible = profileUiState.travelCertificateAvailable,
-    enter = fadeIn() + expandVertically(expandFrom = Alignment.CenterVertically, clip = false),
-    exit = fadeOut() + shrinkVertically(shrinkTowards = Alignment.CenterVertically, clip = false),
-    label = "Travel",
-  ) {
+  if (profileUiState.travelCertificateAvailable) {
```

Comment:
Do we want these to jump in instead of animate in now? Did the existing animation interfere with something else? In any case, this would make these items jump more suddenly in or out of the view right? if yes, is this what we want here?

Reply:
Or is it because we have the placeholders anyway initially, so the moment that this would run here it would be at the same time that we go from placeholder -> real content, so it wouldn't show any jumpiness?